src/styles/blocks/product.scss

```
@@ -0,0 +1,93 @@
+.product {
+  font-size: 10px;
+  width: 166px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 0;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-top: 40px;
+  }
+
+  &__code {
+    line-height: 14px;
+    color: $second-text-color;
+    margin-top: 4px;
+    display: flex;
+    width: 100%;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+  }
+
+  &__reviews-text {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    color: $main-text-color;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__reviews {
    display: flex;
    align-items: flex-end;
    justify-content: space-between;
    
    &-text {
      // write styles here
    }
  }

```